settings/Multisite.setting.php

```diff
@@ -48,7 +48,7 @@
     'is_domain' => 1,
     'is_contact' => 0,
     'description' => ts('Make CiviCRM aware of multiple domains. You should configure a domain group if enabled'),
-    'help_link' => ['page' => 'Multi Site Installation', 'resource' => 'wiki'],
+    'documentation_link' => ['page' => 'Multi Site Installation', 'resource' => 'wiki'],
```

Comment:
@eileenmcnaughton these are giving off 500 errors linking to docs.civicrm.org not the old wiki

Reply:
with the changes to `templates/CRM/Core/Form/Field.tpl` this is now resolved